source/source-connector/configuration-properties/change-stream.txt

```diff
@@ -82,10 +82,19 @@ Settings
      - | **Type:** boolean
        |
        | **Description:**
-       | Whether to publish the changed document instead of the full change
-         stream document. When set to ``true``, the connector
-         automatically sets the ``change.stream.full.document`` property to
-         ``updateLookup`` to receive the updated documents.
+       | Whether to return only the ``fullDocument`` field from the full
+         change stream document. The ``fullDocument`` field contains the
+         most current version of the updated document. When set to
+         ``true``, the connector automatically sets the
+         ``change.stream.full.document`` property to ``updateLookup`` to
+         receive the updated documents.
+       
+       .. important::
+          
+          You must set pipeline configurations against the change stream
+          event, not the ``fullDocument``. To learn how to
+          configure a pipeline, see the ``pipeline`` setting description.
```

Comment:
Issue:

I'm not certain whether this conflicts with my prior suggestion, but I think this info should be called out in the pipeline setting rather than "publish.full.document.only". This setting only determines whether the connector should ignore or apply what's in the "pipeline" setting.

Suggestion:

Happy to pair on a solution in a Zoom meeting if that might work better.

Reply:
moved content to `pipeline` setting description and updated the example to show how how to set configurations using `fullDocument.fieldName`